packages/mapviewer/tests/cypress/tests-e2e/search/search-results.cy.js

```diff
@@ -44,9 +45,49 @@ function checkDescendantOf(selector, message) {
         return $element
     }
 }
+const acceptedDelta = 0.1
+const checkLocation = (expected, result) => {
+    expect(result).to.be.an('Array')
+    expect(result.length).to.eq(2)
+    expect(result[0]).to.approximately(expected[0], acceptedDelta)
+    expect(result[1]).to.approximately(expected[1], acceptedDelta)
+}
+
+function testQueryPositionCrosshairStore({
+    searchQuery,
+    expectedCenter,
+    expectedPinnedLocation,
+    expectedCrosshair = null,
+    expectedCrosshairPosition = null,
+}) {
```

Comment:
In your tests, you're always giving the expected Crosshair value and expected crosshair position. This is good in my opinion as we want test to be explicits as much as possible. I would remove the default values of `expectedCrosshair` and `expectedCrosshairPosition` here, to enforce future possible tests to be explicits.